#**Customer Support Ticket Analyzer**

## **Step 1**: Preloaded Tickets

In [ ]:
# Create a dictionary with the 10 customer tickets
ticket_data = {'Ticket_No': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
               'Customer_Name': [ 'Ravi', 'Meera', 'Sam', 'Anu', 'Rakesh', 'Divya', 'Arjun', 'Kiran', 'Leela', 'Nisha' ],
               'Issue_Description': [ ' Internet not working!!! ', 'slow response, very poor service ',
                                      'GREAT support! issue resolved.',' okay... need help ',
                                       'not BAD but slow', 'Excellent guidance, Very Helpful!',
                                       'good support and good behaviour!', 'Poor handling of technical issue',
                                       'Satisfied. Could be better.', 'Good service... quick response.' ],
              'Priority': ['High', 'Low', 'High', 'Medium', 'Low', 'High', 'Medium', 'High', 'Low', 'Medium']}

print(ticket_data)

{'Ticket_No': [1, 2, 3, 4, 5, 6, 7, 8, 9, 10], 'Customer_Name': ['Ravi', 'Meera', 'Sam', 'Anu', 'Rakesh', 'Divya', 'Arjun', 'Kiran', 'Leela', 'Nisha'], 'Issue_Description': [' Internet not working!!! ', 'slow response, very poor service ', 'GREAT support! issue resolved.', ' okay... need help ', 'not BAD but slow', 'Excellent guidance, Very Helpful!', 'good support and good behaviour!', 'Poor handling of technical issue', 'Satisfied. Could be better.', 'Good service... quick response.'], 'Priority': ['High', 'Low', 'High', 'Medium', 'Low', 'High', 'Medium', 'High', 'Low', 'Medium']}


### **Task:** Print the initial ticket data in a readable format.

In [ ]:
print("\033[1mTicket Details:\033[0m")
for i in range(len(ticket_data['Ticket_No'])):
  print()
  print(f"Ticket_No: {ticket_data['Ticket_No'][i]}")
  print(f"Customer_Name: {ticket_data['Customer_Name'][i]}")
  print(f"Issue Description: {ticket_data['Issue_Description'][i]}")
  print(f"Priority: {ticket_data['Priority'][i]}" )
  print("_ "*30)


Ticket Details:

Ticket_No: 1
Customer_Name: Ravi
Issue Description:  Internet not working!!! 
Priority: High
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

Ticket_No: 2
Customer_Name: Meera
Issue Description: slow response, very poor service 
Priority: Low
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

Ticket_No: 3
Customer_Name: Sam
Issue Description: GREAT support! issue resolved.
Priority: High
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

Ticket_No: 4
Customer_Name: Anu
Issue Description:  okay... need help 
Priority: Medium
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

Ticket_No: 5
Customer_Name: Rakesh
Issue Description: not BAD but slow
Priority: Low
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

Ticket_No: 6
Customer_Name: Divya
Issue Description: Excellent guidance, Very Helpful!
Priority: High
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

Ticket_No: 7
Customer_Name: Arjun
Issue Descr

## **Step 2:** Add More Tickets

In [ ]:
# Ask the user: How many new tickets do you want to add?
# ❖ For each new ticket, collect:
# ➢ Customer Name
# ➢ Issue Description
# ➢ Priority (High / Medium / Low)
New_tickets = int(input("How many new tickets do you want to add: "))

for i in range(New_tickets):
  print(f"\nEnter Details of New Tickets")
# ● Ticket numbers must auto-increment starting from 11.
  print(f"\nTicket_No {11+i}")
  CustomerName=input("Enter Customer Name: ")
  Issue = input("Please enter the issue description: ")
## ● Validate priority (accept only High, Medium, Low).
  while(True):
    Priority_Ticket =input("Enter the priority of Ticket: ").capitalize()
    if(Priority_Ticket in ('High','Medium','Low')):
      break
    else:
      print("\033[1mInvalid input. Please enter High, Medium, or Low.\033[0m")
      continue
# Append new data to ticket_data.
  ticket_data['Ticket_No'].append(11+i)
  ticket_data['Customer_Name'].append(CustomerName)
  ticket_data['Issue_Description'].append(Issue)
  ticket_data['Priority'].append(Priority_Ticket)

print(f"\n{ticket_data}")

How many new tickets do you want to add: 10

Enter Details of New Tickets

Ticket_No 11
Enter Customer Name: Aditya Rao
Please enter the issue description:  Internet disconnects randomly during usage
Enter the priority of Ticket: High

Enter Details of New Tickets

Ticket_No 12
Enter Customer Name: Bhavya Srinivasan
Please enter the issue description: Service response is delayed and not satisfactory
Enter the priority of Ticket: High

Enter Details of New Tickets

Ticket_No 13
Enter Customer Name: Charan Tej
Please enter the issue description: Support team handled the issue efficiently
Enter the priority of Ticket: low

Enter Details of New Tickets

Ticket_No 14
Enter Customer Name: Deepthi Varghese
Please enter the issue description: Need assistance with account verification process
Enter the priority of Ticket: Medium

Enter Details of New Tickets

Ticket_No 15
Enter Customer Name: Eshwar Prasad
Please enter the issue description: Application works fine but performance is slightly sl

#### **Adding extra data to get meaningful insights for the project**

In [ ]:
# Add more tickets data to ticket_data
# Ticket numbers
new_ticket_no = list(range(31,151))


# Customer Names (120)
new_names = [
"Aarav Reddy","Diya Sharma","Kiran Kumar","Nisha Patel","Rohit Verma",
"Ananya Iyer","Vivek Nair","Pooja Singh","Rahul Das","Sneha Gupta",
"Aditya Rao","Bhavya Srinivasan","Charan Tej","Deepthi Varghese","Eshwar Prasad",
"Tejaswini Deshpande","Sai Charitha","Antara Roy","Priyanka Mohan","Nirupam Roy",
"Prithvi Chowhan","Rati Jalan","Dipshika Narayan","Manasa Desaboina","Laxmi Srinivas",
"Divya Parupalli","Mridhula Rajesh","Chitra Ramakrishnan","Lokesh Gowda","Gowri Krishnan",

"Arjun Mehta","Neha Kapoor","Sandeep Yadav","Kavya Reddy","Tarun Gupta",
"Swathi Nair","Rakesh Patel","Meena Iyer","Harsha Kumar","Anjali Verma",
"Praveen Das","Lakshmi Devi","Gopal Krishna","Nikhil Jain","Preeti Singh",
"Suresh Babu","Jyothi Reddy","Mahesh Rao","Anusha Sharma","Ravi Teja",

"Keerthi Nair","Siddharth Gupta","Latha Iyer","Manoj Yadav","Pallavi Singh",
"Ritu Sharma","Ashok Kumar","Savita Devi","Kishore Kumar","Jyothi Nair",
"Abhishek Singh","Manasa Reddy","Gopal Verma","Ramesh Yadav","Shobha Gupta",

"Tejas Patel","Nandini Iyer","Vivek Sharma","Reshma Khan","Imran Ali",
"Farhan Sheikh","Ayesha Begum","Salman Khan","Zoya Fatima","Irfan Khan",
"Sameer Ansari","Nazia Parveen","Shahid Ali","Yusuf Khan","Arif Hussain",

"Sana Sheikh","Faizan Ahmed","Rukhsar Begum","Tanmay Deshpande","Snehal Patil",
"Omkar Kulkarni","Vaishali Jadhav","Sagar Joshi","Pranav Patil","Madhuri Pawar",
"Rohit Desai","Komal Shah","Nitin Mehta","Jay Patel","Krupa Shah",

"Hardik Pandya","Bhavya Trivedi","Chirag Modi","Darshan Solanki","Yash Shah",
"Dhruv Patel","Hiral Mehta","Nirali Shah","Ritu Agarwal","Pankaj Agarwal",
"Neeraj Gupta","Shweta Agarwal","Alok Mishra","Deepak Mishra","Ankita Tiwari",

"Ritu Saxena","Manish Srivastava","Kiran Dubey","Vandana Tripathi","Anurag Shukla",
"Pooja Chaturvedi","Ravi Tripathi","Neha Mishra","Sunil Pandey","Megha Tiwari"
]


# Issue Descriptions (120 UNIQUE, messy + punctuation)
new_issues = [
"internet speed is very low... unable to work properly!!!",
"support team was excellent!!! resolved issue quickly...",
"response time is very high... waited too long!!!",
"good service overall... but improvement needed...",
"poor network quality... frequent disconnections!!!",
"application performance is low... needs optimization...",
"excellent guidance provided!!! very helpful...",
"high delay in response... not satisfied...",
"good support... but issue still persists...",
"poor handling of request... disappointing experience!!!",

"excellent service!!! very quick resolution...",
"low internet stability... connection drops often...",
"good experience... but response was delayed...",
"high loading time... system is very slow!!!",
"poor customer service... no proper updates...",
"excellent support team!!! very professional...",
"low performance in application... frustrating...",
"good assistance provided... satisfied overall...",
"high error rate... system not reliable!!!",
"poor issue resolution... not acceptable...",

"excellent response!!! issue handled quickly...",
"low speed internet... cannot attend meetings...",
"good behaviour from team... but slow response...",
"high delay in processing request... needs fix...",
"poor experience overall... not satisfied!!!",
"excellent help from support!!! very clear explanation...",
"low quality service... needs improvement...",
"good service but performance is low...",
"high waiting time... very frustrating!!!",
"poor system performance... frequent errors...",

"excellent assistance!!! problem solved quickly...",
"low bandwidth issue... affecting work...",
"good support... but follow-up missing...",
"high response delay... needs improvement...",
"poor handling of technical issue... disappointing...",
"excellent experience!!! smooth process...",
"low application speed... not usable...",
"good service overall... but can improve...",
"high latency issue... system slow...",
"poor support response... very slow...",

"excellent team support!!! very helpful...",
"low network strength... unstable connection...",
"good support experience... satisfied...",
"high delay in login process... irritating...",
"poor service quality... not expected...",
"excellent quick response!!! appreciated...",
"low system efficiency... needs upgrade...",
"good behaviour... but issue unresolved...",
"high downtime... system unavailable often...",
"poor customer interaction... not good...",

"excellent service!!! very happy...",
"low signal strength... connection issues...",
"good assistance... but resolution slow...",
"high error frequency... system unstable...",
"poor technical support... not helpful...",
"excellent coordination from team!!!",
"low processing speed... very slow...",
"good support but delay is high...",
"high issue volume... response delayed...",
"poor system handling... disappointing...",

"excellent guidance!!! very clear support...",
"low connectivity issue... frequent drops...",
"good service... but needs improvement...",
"high waiting period... too long...",
"poor response from team... not satisfied...",
"excellent help!!! very quick...",
"low internet quality... unstable...",
"good experience overall... decent...",
"high load time... very slow...",
"poor application behaviour... errors often...",

"excellent support!!! issue fixed...",
"low performance during peak hours...",
"good service but response time high...",
"high delay in support... frustrating...",
"poor issue tracking... no updates...",
"excellent resolution!!! very fast...",
"low server response... lagging...",
"good support experience... okay...",
"high processing delay... system slow...",
"poor experience... not happy...",

"excellent customer support!!! very helpful...",
"low network speed... not usable...",
"good service... but could be better...",
"high downtime issue... needs fix...",
"poor handling... not satisfied...",
"excellent assistance!!! great support...",
"low system response... very slow...",
"good interaction... but delay exists...",
"high issue delay... not acceptable...",
"poor quality support... disappointing...",

"excellent service!!! well handled...",
"low bandwidth... affecting performance...",
"good support team... but slow resolution...",
"high system load... performance drops...",
"poor service experience... bad...",
"excellent help from team!!!",
"low efficiency in system... needs fix...",
"good service but high delay...",
"high error occurrence... unstable...",
"poor response time... too slow...",

"overall performance feels low today... pages taking too long to open!!!",
"excellent experience with support!!! they explained everything clearly...",
"issue priority was high but still took time to resolve... not expected...",
"good interaction with the team... but final solution didn’t work properly...",
"network quality seems poor lately... connection keeps breaking!!!",

"support handled the case in an excellent manner... really impressed...",
"system response is very low during evening hours... affecting work...",
"good service provided, however delay was higher than expected...",
"high latency observed while accessing dashboard... needs attention...",
"technical handling was poor... issue still remains unresolved...",

"excellent coordination from backend team!!! problem fixed smoothly...",
"low efficiency noticed in application flow... takes multiple retries...",
"good behaviour from support staff... but resolution was incomplete...",
"high processing time during transactions... very inconvenient!!!",
"poor experience with recent update... features not working properly...",

"excellent troubleshooting done by team!!! very helpful guidance...",
"low network strength detected... unable to maintain stable connection...",
"good support overall, still waiting for complete resolution...",
"high downtime experienced yesterday... service was unavailable...",
"poor communication from support side... no updates shared..."
]


# Priority (120 values)
new_priority = [
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",

"Medium","High","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",

"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",

"Medium","High","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium",
"High","Medium","Low","Medium","High","Low","Medium","High","Low","Medium"
]


# ==============================
# STEP 3: VALIDATION
# ==============================

print(len(new_ticket_no))
print(len(new_names))
print(len(new_issues))
print(len(new_priority))

120
120
120
120


In [ ]:
# ==============================
# STEP 4: APPEND
# ==============================

ticket_data["Ticket_No"].extend(new_ticket_no)
ticket_data["Customer_Name"].extend(new_names)
ticket_data["Issue_Description"].extend(new_issues)
ticket_data["Priority"].extend(new_priority)


# ==============================
# STEP 5: FINAL CHECK
# ==============================

print("\nFinal lengths:")
for key in ticket_data:
    print(key, len(ticket_data[key]))


Final lengths:
Ticket_No 140
Customer_Name 140
Issue_Description 140
Priority 140


## **Step 3:** Text Cleaning for Issue Descriptions

In [ ]:
# ❖ Clean all issue descriptions by applying:
# ➢ Convert multiple spaces → single space
# ➢ Remove leading/trailing spaces
# ➢ Convert text to lowercase
# ➢ Replace slang/shorthand (example: "ok" → "okay")
# ❖ Hints: Use .replace(), .split(), ' '.join(), .strip(), .lower()
# ➢ Remove punctuation (.,!?-)
for i in range(len(ticket_data['Ticket_No'])):

  ticket = ticket_data['Issue_Description'][i]

  ticket = ticket.replace("!","")
  ticket = ticket.replace(".","")
  ticket = ticket.replace(",","")
  ticket = ticket.strip()
  ticket = ticket.lower()
  ticket = ticket.replace('?','')
  ticket = ticket.replace('-',' ')
  words = ticket.split()
  words = ["okay" if word == "ok" else word for word in words]
  ticket = " ".join(words)

  ticket_data['Issue_Description'][i]=ticket

  print(ticket_data['Issue_Description'][i])

internet not working
slow response very poor service
great support issue resolved
okay need help
not bad but slow
excellent guidance very helpful
good support and good behaviour
poor handling of technical issue
satisfied could be better
good service quick response
internet disconnects randomly during usage
service response is delayed and not satisfactory
support team handled the issue efficiently
need assistance with account verification process
application works fine but performance is slightly slow
very helpful support issue explained clearly
customer service was polite and supportive
unable to login system shows error repeatedly
unable to update profile changes not saved
satisfied overall but service could be better
internet speed is very low unable to work properly
support team was excellent resolved issue quickly
response time is very high waited too long
good service overall but improvement needed
poor network quality frequent disconnections
application performance is low needs o

##**Step 4:** Keyword-Based Issue Insights


In [ ]:
# Create a function def count_tickets_with_word(word):
def count_tickets_with_word(word):
  word=word.lower()
  count=0

  for i in range(len(ticket_data['Ticket_No'])):
    text = ticket_data['Issue_Description'][i]
    if(word in text):
      count+=1
  #Returns how many ticket descriptions contain the given word
  return count

print(f"Number of tickets containing 'poor': {count_tickets_with_word("poor")}")
print(f"Number of tickets containing 'good': {count_tickets_with_word("good")}")
#Case-insensitive search
print(f"Number of tickets containing 'slow': {count_tickets_with_word("sloW")}")
print(f"Number of tickets containing 'excellent': {count_tickets_with_word("Excellent")}")

Number of tickets containing 'poor': 26
Number of tickets containing 'good': 27
Number of tickets containing 'slow': 14
Number of tickets containing 'excellent': 25


In [ ]:
ticket_data["Ticket_No"] = list(range(1, len(ticket_data["Ticket_No"]) + 1))
print(ticket_data["Ticket_No"][-1])

140


##**Step 5:** Final Summary & Insights

###### **1. Display Final Cleaned ticket_data**

In [ ]:
from pprint import pprint

pprint(ticket_data)



{'Customer_Name': ['Ravi',
                   'Meera',
                   'Sam',
                   'Anu',
                   'Rakesh',
                   'Divya',
                   'Arjun',
                   'Kiran',
                   'Leela',
                   'Nisha',
                   'Aditya Rao',
                   'Bhavya Srinivasan',
                   'Charan Tej',
                   'Deepthi Varghese',
                   'Eshwar Prasad',
                   'Tejaswini Deshpande',
                   'Sai Charitha',
                   'Antara Roy',
                   'Priyanka Mohan',
                   'Nirupam',
                   'Aarav Reddy',
                   'Diya Sharma',
                   'Kiran Kumar',
                   'Nisha Patel',
                   'Rohit Verma',
                   'Ananya Iyer',
                   'Vivek Nair',
                   'Pooja Singh',
                   'Rahul Das',
                   'Sneha Gupta',
                   'Aditya Ra

###### **2.Priority Analysis**

In [ ]:
count_high=0
for i in range(len(ticket_data['Ticket_No'])):
  if ticket_data['Priority'][i].capitalize()== "High":
    count_high+=1
print(f"\033[1mNumber of High-priority tickets:\033[0m {count_high}")

Number of High-priority tickets: 43


In [ ]:
count_Medium=0
for i in range(len(ticket_data['Ticket_No'])):
  if ticket_data['Priority'][i].capitalize()== "Medium":
    count_Medium+=1
print(f"\033[1mNumber of Medium-priority tickets:\033[0m {count_Medium}")

Number of Medium-priority tickets: 54


In [ ]:
count_Low=0
for i in range(len(ticket_data['Ticket_No'])):
  if ticket_data['Priority'][i].capitalize()== "Low":
    count_Low+=1
print(f"\033[1mNumber of Low-priority tickets:\033[0m {count_Low}")

Number of Low-priority tickets: 43


###### **3. Find the Ticket With the Longest Issue Description**


In [ ]:
# Creating a empty dictionary
Word_count_dict={}

#Finding the word_count using split and length,then adding to dictionary.
#Dictionary contains index as key,and value as word_count
for i in range(len(ticket_data['Ticket_No'])):
 Word_count_dict[i]= len(ticket_data['Issue_Description'][i].split())

#Finding the max_word_count
max_word_count=max(Word_count_dict.values())

#finding the key corresponding to max_word_count and prinitng the details of it.
for key,value in Word_count_dict.items():
  if value==max_word_count:
    print(f"Ticket_No: {ticket_data['Ticket_No'][key]}")
    print(f"Customer_Name: {ticket_data['Customer_Name'][key]}")
    print(f"Issue Description: {ticket_data['Issue_Description'][key]}")
    print(f"Word Count: {value}" )

Ticket_No: 123
Customer_Name: Nirali Shah
Issue Description: issue priority was high but still took time to resolve not expected
Word Count: 12


In [ ]:
print(len(ticket_data["Ticket_No"]))
print(len(ticket_data["Customer_Name"]))
print(len(ticket_data["Issue_Description"]))
print(len(ticket_data["Priority"]))

140
140
140
140


###### **4. Extract Unique Words Used**


In [ ]:
# Creating a dictionary to store "word as key" and their "count as values".
word_freq={}
unique_words=set()

for issue in ticket_data['Issue_Description']:
  words=issue.split()
  for word in words:
    if word in word_freq:
      word_freq[word]+=1
    else:
      word_freq[word]=1
print("\033[1mWords and their count in issue descriptions:\033[0m")
pprint(word_freq)

Words and their count in issue descriptions:
{'acceptable': 2,
 'accessing': 1,
 'account': 1,
 'affecting': 3,
 'an': 1,
 'and': 3,
 'application': 6,
 'appreciated': 1,
 'assistance': 5,
 'attend': 1,
 'attention': 1,
 'backend': 1,
 'bad': 2,
 'bandwidth': 2,
 'be': 3,
 'behaviour': 5,
 'better': 3,
 'breaking': 1,
 'but': 22,
 'by': 1,
 'can': 1,
 'cannot': 1,
 'case': 1,
 'changes': 1,
 'clear': 2,
 'clearly': 2,
 'communication': 1,
 'complete': 1,
 'connection': 5,
 'connectivity': 1,
 'coordination': 2,
 'could': 3,
 'customer': 4,
 'dashboard': 1,
 'decent': 1,
 'delay': 11,
 'delayed': 3,
 'detected': 1,
 'didn’t': 1,
 'disappointing': 4,
 'disconnections': 1,
 'disconnects': 1,
 'done': 1,
 'downtime': 3,
 'drops': 3,
 'during': 4,
 'efficiency': 3,
 'efficiently': 1,
 'error': 4,
 'errors': 2,
 'evening': 1,
 'everything': 1,
 'excellent': 25,
 'exists': 1,
 'expected': 3,
 'experience': 11,
 'experienced': 1,
 'explained': 2,
 'explanation': 1,
 'fast': 1,
 'features': 1,


In [ ]:
# Finding the unique words in the created dictionary and adding it to "unique_words" set
for unique_word,count in word_freq.items():
  if count==1:
    unique_words.add(unique_word)

#Display the count of unique words and sorted unique word lists
print(f"\033[1mCount of all unique words:\033[0m {len(unique_words)}")


Count of all unique words: 115


In [ ]:
word_list=sorted(unique_words)
print("\033[1mSorted word_list\033[0m")
pprint(word_list)

Sorted word_list
['accessing',
 'account',
 'an',
 'appreciated',
 'attend',
 'attention',
 'backend',
 'breaking',
 'by',
 'can',
 'cannot',
 'case',
 'changes',
 'communication',
 'complete',
 'connectivity',
 'dashboard',
 'decent',
 'detected',
 'didn’t',
 'disconnections',
 'disconnects',
 'done',
 'efficiently',
 'evening',
 'everything',
 'exists',
 'experienced',
 'explanation',
 'fast',
 'features',
 'feels',
 'final',
 'fine',
 'flow',
 'follow',
 'for',
 'frequency',
 'higher',
 'however',
 'impressed',
 'improve',
 'incomplete',
 'inconvenient',
 'irritating',
 'issues',
 'keeps',
 'lagging',
 'lately',
 'loading',
 'maintain',
 'manner',
 'meetings',
 'missing',
 'multiple',
 'needed',
 'noticed',
 'observed',
 'occurrence',
 'open',
 'optimization',
 'pages',
 'peak',
 'period',
 'persists',
 'polite',
 'priority',
 'professional',
 'profile',
 'proper',
 'randomly',
 'rate',
 'really',
 'recent',
 'reliable',
 'remains',
 'repeatedly',
 'resolve',
 'retries',
 'satisfact

#**Insights**

###
* A total of 140 tickets indicate frequent customer interaction with support systems.

* **Major issues observed:**
  * Slow system performance and delays,
  * Login and account access problems,
  * Payment failures and transaction errors,
  * Network instability and disconnections.

* Feedback also includes positive mentions like good, excellent, indicating mixed customer sentiment.

* **Performance issues** are likely due to high system load and inefficient backend processing.
* **Login failures** indicates authentication or session handling issues.
* **Payment failures** suggests unreliable gateway or incomplete transaction validation.
* **Repeated complaints** shows lack of root cause resolution.
* **Delays in response** indicates inefficient ticket handling or prioritization.

* Optimize system performance and backend processing.
* Fix login and payment reliability issues.
* Reduce delays through better ticket prioritization.
* Address root causes to prevent recurring issues.
* Improvements will lead to fewer complaints and better customer experience

* **Final conclusion**: Support quality is good, but technical issues and delays impact customer satisfaction.
* Improving the performance, reliability, and resolution speed of tickets will enhance overall service quality.



